index.rst

```diff
@@ -6,24 +6,474 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
-
-   **This technote is a work-in-progress.**
+.. Make in-text citations with: :cite:`bibkey`.
+.. Uncomment to use citations
+.. .. rubric:: References
+.. 
+.. .. bibliography:: local.bib lsstbib/books.bib lsstbib/lsst.bib lsstbib/lsst-dm.bib lsstbib/refs.bib lsstbib/refs_ads.bib
+..    :style: lsst_aa
 
 Abstract
 ========
 
-Principles, guidelines, conventions, formatting style, and workflow for tutorials developed by the Community Engagement Team (CET).
+A guideline for anyone creating or updating tutorials for the Rubin science community.
+This includes all community-facing demonstrations of how to use the Rubin Science Platform or how to analyze LSST data.
+
+Pedagogical principles
+======================
+
+All tutorial developers should adhere to the following principles.
+
+Inclusivity
+-----------
+
+Avoid offensive or exclusionary language at all times (e.g., violent or ableist terms).
+Ensure jargon and acronyms are defined.
+
+Level-appropriate
+-----------------
+
+Tutorials should clearly identify and teach to their target audience (beginner, intermediate, or advanced).
```

Comment:
Perhaps the three levels of skill should be after a colon and not in parentheses?

Reply:
Done